dali/c_api/c_api_test.cc

```diff
@@ -757,21 +757,21 @@ TEST(CApiTest, GetBackendTest) {
                           .AddArg("device", "cpu")
                           .AddArg("name", es_cpu_name)
                           .AddOutput(es_cpu_name, "cpu"), es_cpu_name);
-  std::string decoder_name = "decoder";
-  pipe.AddOperator(OpSpec("decoders__Image")
+  std::string cnt_name = "decoder";
```

Comment:
Nitpick:
```suggestion
  std::string cont_name = "contiguous";
```
It's no longer a "decoder" and `cnt` sounds a bit too much like `count`.

Reply:
Done